test/test_team.py

```diff
@@ -275,16 +278,16 @@ def trackingBot(bot, state):
     state = setup_game(team, max_rounds=300, layout_dict=parse_layout(layout))
     while not state['gameover']:
         state = play_turn(state)
+        # check at the end of every round only, where all the counts should be updated
+        if state['turn'] != 4: continue
```

Comment:
This is `if True: continue` ;)

Reply:
Btw. I did’t see you were also working on this. So now I have a fixed version of this as well.

Reply:
my comment below was meant as an answer to this comment. Just github clearly does not know how to attach email replies to the right thread :(